### PR TITLE
Added support for `Configuration#setTransactional()` configuration in `doctrine/doctrine-migrations-bundle`

### PR DESCRIPTION
**Pull request by Ocramius**
This patch introduces a way to set `Configuration#setTransactional()` via DIC parameters, introduced
in https://github.com/doctrine/migrations/pull/1157, but never implemented at configuration level.

Ref: https://github.com/doctrine/migrations/pull/1157
Ref: https://github.com/doctrine/migrations/issues/1153

@greg0ire note: I've set the milestone to `3.2.0`, but beware that the current default branch is `3.1.x` - unsure why

### CODE REVIEW

File: Tests/Fixtures/conf.xml [hunk 1]
```diff
@@ -2,16 +2,16 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xmlns:doctrine="http://symfony.com/schema/dic/doctrine/migrations/3.0"
            xsi:schemaLocation="
-           http://symfony.com/schema/dic/doctrine/migrations/3.0 http://symfony.com/schema/dic/doctrine/migrations/doctrine_migrations-3.0.xsd
-           http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd
+           http://symfony.com/schema/dic/doctrine/migrations/3.0 ../../Resources/config/schema/doctrine_migrations-3.0.xsd
```
Comment (Ocramius) [inline Tests/Fixtures/conf.xml:5]: This was referencing a random internet URL - fixed via local reference to locked file reference

File: Tests/Fixtures/conf.xml [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+           http://symfony.com/schema/dic/services ../../vendor/symfony/dependency-injection/Loader/schema/dic/services/services-1.0.xsd
```
Comment (Ocramius) [inline Tests/Fixtures/conf.xml:6]: This was referencing a random internet URL - fixed via local reference to locked dependency reference, which will lead to `composer.json` updates producing green/stable XSD validation run

**Comment by Ocramius** [comment]
Whoops, seems like this was already handled by https://github.com/doctrine/DoctrineMigrationsBundle/pull/432 (unreleased), hence why I repeated the effort.

**Comment by Ocramius** [comment]
@greg0ire closing as `duplicate` here: who should trigger the `release-on-milestone-closed.yml`? Should I do that?

**Comment by greg0ire** [comment]
Sorry, I missed @alshenetsky's [comment](https://github.com/doctrine/DoctrineMigrationsBundle/pull/432#issuecomment-909158941), probably a case of holidays and then I came back and cleared my inbox :sweat_smile:

**Pull request closed without merging**